src/main/java/org/jetbrains/bsp/bazel/server/Server.java

```diff
@@ -49,7 +47,8 @@ public static void main(String[] args) {
       System.setOut(logStream);
       System.setErr(logStream);
 
-      BazelBspServer bspServer = new BazelBspServer(bazelPath);
+      BazelBspServerConfig config = BazelBspServerConfig.from(args);
```

Comment:
In order to not have to add this values manually to the json file created with the startup options, we we probably should put these ourselves in the json file. Wdyt?

Reply:
I'm not sure to understand, are you talking about the json generated by `org.jetbrains.bsp.bazel.Install` ?
Do you think that we should add to `argv` the target of the current folder ? I guess that could be done (either via a bazel query of by finding the path up to a WORKSPACE file)

Reply:
I think it means we could pass the desired targets to the `Install` arguments and in turn pass them into `argv`

Reply:
Yeah, as @jastice said :)

Reply:
I've just added the `--targets` flag on the `Install`

Reply:
Looks great! Thanks!